cmd/cronosd/cmd/root.go

```diff
@@ -85,11 +90,30 @@ func NewRootCmd() (*cobra.Command, ethermint.EncodingConfig) {
 				return err
 			}
 
-			initClientCtx, err = config.ReadFromClientConfig(initClientCtx)
+			initClientCtx, err = clientcfg.ReadFromClientConfig(initClientCtx)
```

Comment:
```suggestion
			initClientCtx, err = clientcfg.ReadDefaultValuesFromDefaultClientConfig(initClientCtx)
```

Reply:
reverted, this will override the command flags.

Reply:
sry, it's the 2nd one